src/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -45,6 +47,10 @@ const AdminDashboard = () => {
             getUserDetails()
             dispatch(getRooms(t))
         }
+
+        // return () => {
+        //     dispatch(newRoom())
```

Comment:
remove comment?

Reply:
why did you comment. If you change page you keep previous state and if you come back on admin dashboard you'll have previous message displayed

Reply:
uncommented already, it was a push mistake